openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -68,6 +69,67 @@ OpenAssessment.BaseView.prototype = {
     },
 
     /**
+     * Checks the rendering status of the views.
+     * @returns {boolean} true if any step's view is still loading.
```

Comment:
Learner-facing views? I note that messageView, leaderboardView, and staffAreaView are not listed here.

If new views are added, it would be helpful to understand the contract of this method, so as to know whether or not the new views should be added here as well.